### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by bobmyhill**
Removed support for Python versions which have reached end of life (3.5 and earlier).

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ Authors (as of 2015, listed alphabetically by first name):
 
 ## Requirements
 
-* Python 2.7.x or Python 3.4+
+* Python 3.6+
 * Python modules:
-  NumPy, SciPy, Matplotlib
+  NumPy, SciPy, SymPy, Matplotlib, pycddlib
```
Comment (tjhei) [inline Readme.md:34]: would it make sense to split between required and optional or do you think we should always require pycddlib?

Comment (bobmyhill) [reply Readme.md:34]: We can make pycddlib optional if you want. On a related note, your tester doesn't yet have pycddlib installed...

**Comment by bobmyhill** [comment]
Ok, I've added an Optional category, and added pycddlib to it.

File: Readme.md (hunk 1 above)
Comment (tjhei) [reply Readme.md]: We can just remove my jenkins instance, I would think. Objections?

**Pull request merged**